src/lib/Toast/Toast.svelte

```diff
@@ -0,0 +1,159 @@
+<script lang="ts">
+  import { fly } from 'svelte/transition';
+  import { defaultToastProperties, type ToastProperties } from './properties';
+  import errorIcon from './error.svg';
+  import closeIcon from './close.svg';
+  import infoIcon from './info.svg';
+  import { createEventDispatcher, onDestroy, onMount } from 'svelte';
+
+  export let properties: ToastProperties = defaultToastProperties;
+
+  enum ToastType {
```

Comment:
no need to create a new enum here, you can use already created type ToastType which was exported in properties.ts